docs/topics/basic-syntax.md

```diff
@@ -182,24 +225,24 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-basic-syntax-variable-top-level"}
 
-See also [Properties](properties.md).
+For information about declaring properties, see [Properties](properties.md).
 
 ## Creating classes and instances
 
-To define a class, use the `class` keyword.
+To define a class, use the `class` keyword:
 ```kotlin
 class Shape
 ```
 
-Properties of a class can be listed in its declaration or body. 
+Properties of a class can be listed in its declaration or body: 
 
 ```kotlin
 class Rectangle(val height: Double, val length: Double) {
     val perimeter = (height + length) * 2 
 }
 ```
 
-The default constructor with parameters listed in the class declaration is available automatically. 
+The default constructor with parameters listed in the class declaration is available automatically:
 
 ```kotlin
 class Rectangle(val height: Double, val length: Double) {
```

Comment:
I know this was here, and you changed nothing here, but I feel like since we are talking about primary constructors it might help to show the entire code, not just the main function.
so I suggest removing sampleStart sampleEnd from this code